EIPS/eip-7742.md

```diff
@@ -57,43 +61,78 @@ the blob max is desirable to reduce rigidity in this protocol parameter.
 Even if the EL keeps a fixed target value based on the max, removing the max implies the EL would not know what the target
 value should be. To address this lack of information, this EIP proposes the CL sends the current target value to the EL
 with each provided payload over the Engine API. The EL block header will also need to be extended with this target value
-to preserve the security of optimistic sync.
+(`target_blobs_per_block`) to preserve the security of optimistic sync.
+
+#### Normalized `excess_gas`
+
+With the changing target, we also need to scale the `BLOB_BASE_FEE_UPDATE_FRACTION` from [EIP-4844](./eip-4844.md)
+accordingly to bound the price jumps by +-12.5%. But this introduces some irregularity whenever target gets updated because
+excess blob gas (accumulated via old target) is now effectively scaled down by this changed fraction in the fee calculations.
+
+To mitigate this irregularity, we now save a _normalized_ excess gas with respect to a fixed
+`EXCESS_BLOB_GAS_NORMALIZATION_TARGET` and use a corresponding `BLOB_BASE_FEE_UPDATE_FRACTION_NORMALIZED` in the base fee
+calculations.
 
 ## Specification
 
-### Block structure and validity
+| Constant | Value | Remarks|
+| - | - | - |
+| `OLD_TARGET_BLOBS_PER_BLOCK` | `3` | blob target corresponding to [EIP-4844](./eip-4844.md) |
```

Comment:
nitpick, but I think better to call this `TARGET_BLOBS_PER_BLOCK_4844` as "old" is a relative term and may not age as well across protocol development